kubernetes/annotations_and_labels.md

```diff
@@ -28,6 +28,19 @@
   (i.e. networking, DNS, monitoring, ingress controller, etc.). Latter would be
   managed by `chart-operator`.
 
+## Common Labels (for guest cluster nodes)
+
+- `release.giantswarm.io/version` - value should be the used Giant Swarm release
+  version, e.g. `2.3.0`. Should be applied to all Kubernetes nodes, regardless
+  of their role.
+- `NAME.operator.giantswarm.io/version` - value should be the used operator
```

Comment:
I added another level for the operator name. My idea is that you can compute the provider operator's version by taking the provider label and read the version label for the provider operator. This makes also more sense for the label structure since there are multiple operators and we should be specific here, just like we made it more straight forward with the provider label.

Reply:
So in that case I'd say we should stick to general naming scheme: https://github.com/giantswarm/fmt/blob/master/kubernetes/annotations_and_labels.md#naming
It would just be `kvm-operator.giantswarm.io/version`

Reply:
I don't like this, but I follow now.

Reply:
Done.

Reply:
I also think it's ok to have both if that is useful.

```
kvm-operator.giantswarm.io/version
operator.giantswarm.io/version
```

But having something `kvm-operator.operator.giantswarm.io/version` or `kvm.operator.giantswarm.io/version` feels just weird or it's so close to convention we have that I don't see a reason to break it.